src/locales/en-GB.ts

```diff
@@ -60,7 +60,7 @@ export default {
   },
 
   TRANSACTION: {
-    AMOUNT: "Amount",
+    AMOUNT: "UNS Flow",
```

Comment:
i count 8 use of this variable. each time for transactions amount.
The only use wich is strange is "UNS Flow" instead of amount in the QRcode popup... i gonna change this one!

Reply:
i need translations for every supported langages ^^